README.md

```diff
@@ -39,10 +39,11 @@ stage('Publish') {
             ownerName: 'janes-addiction',
             appName: 'ritual-de-lo-habitual',
             pathToApp: 'three/days/xiola.apk',
-            distributionGroups: 'casey, niccoli'
+            distributionGroups: 'casey, niccoli',
+            notifyTesters: true
```

Comment:
I wanted to limit this example to the simplest form of using the plugin. In which case this flag would default to true as discussed.

I think you're right in that we should provide examples of other usages but can we do that elsewhere please.

Reply:
Sure thing. I'll revert this

Reply:
I've raised [JENKINS-59692](https://issues.jenkins-ci.org/browse/JENKINS-59692) to fill in examples.